ciao-cli/image.go

```diff
@@ -350,6 +351,87 @@ func uploadTenantImage(username, password, tenant, image, filename string) error
 	return res.Err
 }
 
+type imageModifyCommand struct {
+	Flag       flag.FlagSet
+	name       string
+	imageID    string
+	visibility string
+	tags       string
+}
+
+func (cmd *imageModifyCommand) usage(...string) {
+	fmt.Fprintf(os.Stderr, `usage: ciao-cli [options] image set [flags] imageID
+
+Modify an image
+
+The modify flags are:
```

Comment:
You are calling the command 'set' on line 75 so it should be named set here too.
Or name it modify if it makes more sense